test/integration/targets/openssl_certificate_info/tasks/main.yml

```diff
@@ -27,7 +27,7 @@
       GN: First Name
       title: Chief
       pseudonym: test
-      UID: asdf
+      x500UniqueIdentifier: asdf
```

Comment:
That would have been a "UserID" (OID `0.9.2342.19200300.100.1.1`) before - I guess this change is intentional?

Reply:
It was, though I'm not sure whether it is still necessary (or whether that problem was caused by lowercasing). I'll add a commit to revert this, let's see what happens.

Reply:
Looks like it isn't necessary anymore. I've removed the same change from #54921.